### PR TITLE
fixed vpc rename bug

### PR DESCRIPTION
**Pull request by ddegoede**
Hi Sander,

renaming a VPC does not work, I think this will fix it, but I could not test it, 'make dev' does not work for me at the moment.

Daan

### CODE REVIEW

**Comment by ddegoede** [comment]
@svanharmelen  BTW, I see that my master is ahead of yours.. thus the large amount of changed files.

File: builtin/providers/cloudstack/resource_cloudstack_vpc.go [hunk 1]
```diff
@@ -187,14 +187,19 @@ func resourceCloudStackVPCUpdate(d *schema.ResourceData, meta interface{}) error
...
+		name := d.Get("name").(string)
+
 		// Set the display text
 		displaytext, ok := d.GetOk("display_text")
 		if !ok {
-			displaytext = d.Get("name")
+			displaytext = name
 		}
 		// Set the (new) display text
 		p.SetDisplaytext(displaytext.(string))
```
Comment (svanharmelen) [inline builtin/providers/cloudstack/resource_cloudstack_vpc.go]: This will give an error when `displaytext` is not an `interface{}`. So if we set `displaytext` to be the `name` (which is a `string` here), this will give an error.

So I would revert this change and only add one line (on line 190):

```go
p := cs.VPC.NewUpdateVPCParams(d.Id())

// Set the (new) name
p.SetName(d.Get("name").(string)

<snip>
```

**Comment by svanharmelen** [comment]
@ddegoede nice find! The solution needs some attention (see comment), but it should be fairly easy to update the PR...

Additionally you should not make this PR against my fork, but against the source repo from HashiCorp :wink:

**Comment by svanharmelen** [comment]
Fixed in the source repo: https://github.com/hashicorp/terraform/pull/8784
